scripts/linters/test_files/valid_ignored.service.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview An Injectable, that is not present in Angular Services Index
+ * js_ts_linter_test.py. This file contains an Injectable.
+ */
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class MockCsrfTokenService< extends SomeService {}
```

Comment:
What is the "<"?

Reply:
It was to cover the lines when I was manually parsing the class name. I have removed it. With the regex this isn't any problem.